src/main/java/at/porscheinformatik/weblate/spring/WeblateMessageSource.java

```diff
@@ -155,7 +156,6 @@ public void reload(Locale... locales) {
     translations = new HashMap<>(loadTranslation(new Locale(locale.getLanguage())));
 
     if (StringUtils.hasText(locale.getCountry())) {
-      Map<String, String> countrySpecific = loadTranslation(locale);
```

Comment:
Why remove this? This loads the country-specific translations and overrides the ones with only language